examples/GUI/index.html

```diff
@@ -49,7 +49,7 @@ <h1>Online SQL interpreter</h1>
 
     <button id="execute" class="button">Execute</button>
     <button id='savedb' class="button">Save the db</button>
-    <label class="button">Load an SQLite database file: <input type='file' id='dbfile'></label>
+    <label class="button">Load a SQLite database file: <input type='file' id='dbfile'></label>
```

Comment:
I think both are correct... The creator of SQLite says "an SQLite database"...

https://en.wikipedia.org/wiki/SQLite
https://english.stackexchange.com/questions/431329/what-is-the-correct-pronunciation-of-sqlite

Reply:
ok reverted grammar-changes in commit c6aa8884baa921e3aedf31f30796db1ddece5c35